addons/survey/static/src/views/fields/boolean_toggle/survey_boolean_toggle_field.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module **/
+import { registry } from "@web/core/registry"
+import { BooleanToggleField } from '@web/views/fields/boolean_toggle/boolean_toggle_field';
+
+export class SurveyBooleanToggleField extends BooleanToggleField {
```

Comment:
Commit message does not explain "why" we have to do this... should contain the technical explanation as it is not obvious why we have to do this :)